docs/source/whatsnew/v0.4.2.txt

```diff
@@ -0,0 +1,4 @@
+Bug Fixes
```

Comment:
No need for the entry here...this is not actually a bug (no unexpected/incorrect behavior). Can leave this empty for now.

Reply:
Should I delete this file for now or keep it?